### DISCUSSION TITLE
Multi-Image Upload for Galleries?

### DISCUSSION THREAD

**Discussion by daniel-granger** [sveltia/sveltia-cms#189] [Q&A] [answered]
I'm looking to migrate a site away from Netlify CMS to Sveltia, and was wondering if multi-image uploads were supported for galleries (I know you can multi-upload to the assets which is great)

Currently, Sveltia appears like this:
![image](https://github.com/user-attachments/assets/26c0ec5d-5926-45af-8929-836d8a84162d)

with the yaml being defined as:
`- { label: "Gallery", name: "galleryImages", widget: "list", summary: '{{fields.image}}', field: {label: Image, name: image, widget: image} }`

Would this be something that's easily achievable? Ideally without any 3rd party image providers.

Ta.

**Comment by kyoshino** [comment, marked as answer]
Multiple image selection is not supported yet, but I do have a plan to implement it. There is a feature request issue here: #10. Will work on it once the ongoing Svelte 5 migration is complete.
